pkg/maps/maps.go

```diff
@@ -39,3 +39,8 @@ func Values[M ~map[K]V, K comparable, V any](m M) []V {
 func Keys[M ~map[K]V, K comparable, V any](m M) []K {
 	return maps.Keys(m)
 }
+
+// Merge returns
```

Comment:
Could you copy & paste the comment from the maps package? I think it's clearer than this one.

Reply:
Thanks, It has revised